hybrid/compartment_examples/comp_setup/main.c

```diff
@@ -43,13 +42,23 @@ const size_t total_comp_size = 5000;
 size_t id = 0;
 const size_t comp_cnt = 1;
 
-// For each compartment, we will store to memory:
-// * an id
-// * the ddc corresponding to the compartment
-// * the function within the compartment to be executed upon switching
+/* Abstract representation of a compartment. Within 80 bytes we represent:
```

Comment:
I'm not sure we want to put the sizes in here since they're not really guaranteed. FWIW, I think the size of the struct is fine and we don't have to worry about it!

Reply:
The sizes are more for me so I can easily calculate the offsets in `shared.S` without needing to look at memory layout in `gdb` (as well as an implicit explanation of why the offset are as they are in that file).